dex/networks/eth/params.go

```diff
@@ -242,9 +242,6 @@ var testTokenID, _ = dex.BipSymbolID("dextt.eth")
 var usdcTokenID, _ = dex.BipSymbolID("usdc.eth")
 
 // Gases lists the expected gas required for various DEX and wallet operations.
-// ★★ IMPORTANT ★★  By policy, clients should allow servers to adjust Swap and
-// Redeem gas values in their *dex.Asset up to but not over 2x the value listed
-// in VersionedGases or Tokens.
```

Comment:
No?

Reply:
No, the `dex.Asset` doesn't even make it into the wallet backends anymore:

https://github.com/decred/dcrdex/commit/f950443754abe7aa302404590be3476a3bfbbbf5
https://github.com/decred/dcrdex/pull/1866#issuecomment-1270331373

Server can specify fee rates, but not gas caps to use the in transactions.  That would be like telling the BTC backend what size in bytes when planning the transaction.  The client is in charge of authoring transactions the way they think is right.

Reply:
We actually have the size fields deprecated from `msgjson.Asset` now: https://github.com/decred/dcrdex/blob/f950443754abe7aa302404590be3476a3bfbbbf5/dex/msgjson/types.go#L1068-L1073